package/plato/postrm

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Opkg doesn't seem to remove that.
```

Comment:
is the newly created symlink preventing removal of this dir? (because opkg wouldn't know to remove the symlink because opkg didn't add it). i'm curious if you can check the symlink into the package? (or if that's technically impossible)

Reply:
Which symlink do you mean? I have defined 3. But this is not a link itself. So even if it contains some symlinks itself, I don't see the problem why removing shouldn't work. Symlinks are just files, but I'm not sure whether they may affect opkgs checks.

Symlinks are:
- `/opt/bin/plato` -> `/opt/lib/plato/plato.sh`
- `/opt/lib/plato/media` -> `/home/root/plato-media/`
- `/opt/lib/plato/Settings.toml` -> `/opt/etc/plato.toml`

Reply:
I'm assuming that opkg has logic to only remove empty directories, so if you add a symlink inside `/opt/lib/plato/`, it will prevent `/opt/lib/plato` from being removed. 

Looking at opkg source (to validate assumption), I see this special logic: https://git.yoctoproject.org/cgit/cgit.cgi/opkg/tree/libopkg/opkg_remove.c#n110

Reply:
my tentative recommendation is to check if your symlinks still exist and if they are syminks. if so, remove the symlinks. otherwise, leave the whole dir.